jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketFrameHandler.java

```diff
@@ -81,10 +80,10 @@ public class JavaxWebSocketFrameHandler implements FrameHandler
     public JavaxWebSocketFrameHandler(JavaxWebSocketContainer container,
                                       UpgradeRequest upgradeRequest,
                                       Object endpointInstance,
-                                      MethodHandle openHandle, MethodHandle closeHandle, MethodHandle errorHandle,
+                                      MethodHolder openHandle, MethodHolder closeHandle, MethodHolder errorHandle,
                                       JavaxWebSocketMessageMetadata textMetadata,
                                       JavaxWebSocketMessageMetadata binaryMetadata,
-                                      MethodHandle pongHandle,
+                                      MethodHolder pongHandle,
                                       EndpointConfig endpointConfig)
```

Comment:
This class is not internal and has a signature change. Is that a problem? If not, can be made internal?

Reply:
Anything in the websocket common modules are not part of the API and considered internal implementation.